python/BioSimSpace/Align/_align.py

```diff
@@ -1785,7 +1804,7 @@ def _validate_mapping(molecule0, molecule1, mapping, name):
     """
 
     for idx0, idx1 in mapping.items():
-            if type(idx0) is int and type(idx1) is int:
+            if isinstance(idx0, int) and isinstance(idx1, int):
```

Comment:
We use `type` when checking for `int` since `isinstance` will also return `True` for `bool` objects.

Reply:
Wow, thanks! I did not know that. 🤣